### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by emmiiorji**
This change creates a basic command line user interface for the application.

The following apply:

- There's an `app.rb` file that serves as console app entry-point. It has methods that do the following:
     - List all books.
     - List all people.
     - Create a person (teacher or student, not a plain Person).
     - Create a book.
     - Create a rental.
     - List all rentals for a given person id.
- There's a `main.rb` file having a main method that:
      - Presents the user with a list of options to perform.
      - Lets users choose an option.
      - When needed, asks for parameters for the option.
      - Can quit the app.

### CODE REVIEW

File: models/app.rb [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      return
+    end
+
+    selected_person = @persons.filter { |person| person.id == person_id }
+    if selected_person.empty?
+      puts "No one with ID #{person_id}."
+      return
+    end
+    puts "Rentals by #{selected_person[0].name}:"
+    rentals_by_person = rentals.filter do |rental|
```
Comment (danmainah) [inline models/app.rb]: - [ ] Kindly check your code, I am running into the error below when I try to check rentals from particular person:

image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @emmiiorji , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: models/app.rb (hunk 1 above)
Comment (emmiiorji) [reply models/app.rb]: Hi @danmainah,

Thanks for the kind review. I've fixed the issue.

**Review by roshan-bajgain — APPROVED** [review]
Hi @emmiiorji, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**